dashboard/config/routes.rb

```diff
@@ -79,12 +79,24 @@ module OPS
   get '/u/:id', to: redirect('/c/%{id}')
   get '/u/:id/:action_id', to: redirect('/c/%{id}/%{action_id}')
 
+  # These links should no longer be created (August 2017), though we will continue to support
+  # existing links. Instead, create /r/ links.
```

Comment:
Judging by your responses above, it does seem we are ok with this 👍 

So the intent here is mainly to disable the `/r/` share link, as stated, when the user is deleted, but not to fully remove the shared code from being publicly accessible.